### DISCUSSION TITLE
How to plot using multiple devices?

### DISCUSSION THREAD

**Discussion by DevilDog08** [swar/Swar-Chia-Plot-Manager#1045] [Q&A]
I have a pc running swar manager for plotting and farming. I have built a new pc for plotting. I would like to know how to plot in the new pc and then move the plot drive to my farming pc.

Please help me to achieve this. I would like to know how to setup swar in new device with the same private key and plot.

Appreciate your help. Thanks

**Comment by FlyingToto** [comment]
note sure if this is the best way, but this seems to work for me... 

1. install chia on your new computer but when prompted enter the key mnemonics that you saved from your first computer.
2. stop chia.
3. install swar on the new computer and configure it the way you want for that new computer only.
4. (not 100% sure if this required) in the config list the public farm and pool keys  from your initial install of chia on your first machine (chi key show)
5. start swar on the new computer, run the view test ui if you want.
6. start chia  (disable upnp if needed)

now in this configuration, you end up running the full node in 2 separate places which is supposedly overkill but it looks like it is working. The official way to setup multiple harvester with 1 node is here: https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
